website/static/vendor/knockout-sortable/knockout-sortable.js

```diff
@@ -351,7 +351,8 @@
         },
         connectClass: ko.bindingHandlers.sortable.connectClass,
         options: {
-            helper: "clone"
+            helper: "clone",
+            tolerance: "pointer"
```

Comment:
Ah, I just noticed that this is an edit to knockout-sortable. In general, we don't want to edit 3rd-party dependencies directly. Instead use configuration/extension when possible. @leb2dg Is this change necessary?